src/main.js

```diff
@@ -1 +1,8 @@
-console.log('Hello, world!');
+// console.log('Hello, world!');
+const accs = Array.from(document.querySelectorAll('.accordion'));
+// console.log(accs);
+function newFunction() {
+  this.classList.toggle('accordion--active');
```

Comment:
Nice start – yes, this opens the clicked accordion.

But, _"Only one accordion item can be shown at a time"_ was required.

So, in [pseudocode](https://en.wikipedia.org/wiki/Pseudocode) here's what you need to do in this function:

1. Check if this accordion item is active.
2. If active, just deactivate to close it.
3. If not active, close the other accordions then activate self.